### DISCUSSION TITLE
There is a typo in documentation

### DISCUSSION THREAD

**Discussion by badrelmazaz** [minio/minio#14243] [Q&A] [answered]
There is a small typo in https://docs.min.io/docs/python-client-api-reference, there is condig instead of config, I dont know where to signal it so sorry if I am posting it in the wrong place.

![image](https://user-images.githubusercontent.com/59953925/152325245-36086c82-59e6-44d7-9a54-84cf8bbcea48.png)

**Comment by harshavardhana** [comment, marked as answer]
It should be fixed here https://github.com/minio/minio-py - thanks for reporting @badr-elmazaz
